debezium-core/src/main/java/io/debezium/relational/RelationalChangeRecordEmitter.java

```diff
@@ -64,7 +66,8 @@ protected void emitCreateRecord(Receiver receiver, TableSchema tableSchema)
         Object[] newColumnValues = getNewColumnValues();
         Object newKey = tableSchema.keyFromColumnData(newColumnValues);
         Struct newValue = tableSchema.valueFromColumnData(newColumnValues);
-        Struct envelope = tableSchema.getEnvelopeSchema().create(newValue, getOffset().getSourceInfo(), getClock().currentTimeAsInstant());
+        Instant instant = getInstantFrom(getOffset());
```

Comment:
That seems not right? The envelope ts_ms is the time of processing in Debezium, not the time of the change in the soure database.

Reply:
I also think this could be what is causing the problems with the PostgreSQL connector.  There is one test in particular that seems to hang indefinitely on Travis CI.  I'm going to revert this locally & check if that fixes PostgreSQL.